relayer/src/services/relayer/optimism.ts

```diff
@@ -86,7 +86,6 @@ export class OptimismRelayerService extends BaseRelayerService {
     await this.l2RpcProvider.waitForTransaction(l2TransactionHash, 1);
     const message = await this.getMessageInTransaction(l2TransactionHash);
     const status = await this.messenger.getMessageStatus(message);
-    console.log(message, status);
```

Comment:
Can you squash this hunk with the previous commit, that adds this? The overall change is a no-op, so this is just noise.